app/data/test_final_confirmation.json

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
Having a smaller schema for this test will emphasis the final confirmation configuration when someone comes to create it for real and it's also just going to be more questions than needed for testing